Source/pack.h

```diff
@@ -13,66 +15,66 @@ namespace devilution {
 
 #pragma pack(push, 1)
 struct PkItemStruct {
-	Uint32 iSeed;
-	Uint16 iCreateInfo;
-	Uint16 idx;
-	Uint8 bId;
-	Uint8 bDur;
-	Uint8 bMDur;
-	Uint8 bCh;
-	Uint8 bMCh;
-	Uint16 wValue;
-	Uint32 dwBuff;
+	int32_t iSeed;
+	uint16_t iCreateInfo;
+	uint16_t idx;
+	uint8_t bId;
+	uint8_t bDur;
+	uint8_t bMDur;
+	uint8_t bCh;
+	uint8_t bMCh;
+	uint16_t wValue;
+	int32_t dwBuff;
 };
 
 struct PkPlayerStruct {
-	Uint32 dwLowDateTime;
-	Uint32 dwHighDateTime;
-	Sint8 destAction;
-	Sint8 destParam1;
-	Sint8 destParam2;
-	Uint8 plrlevel;
-	Uint8 px;
-	Uint8 py;
-	Uint8 targx;
-	Uint8 targy;
+	int32_t dwLowDateTime;
+	int32_t dwHighDateTime;

```

Comment:
And here (won't comment more, seems to be quite a few).

Reply:
Yeah, I never understand why you comment on every instance of general issues.

Reply:
To make sure the person is aware of all instances. It is very common when I perform reviews for the person to only fix the problem on that specific line and forget other similar issues.

Reply:
Ok, I usually update my first comment to mention that they need to go over it and fine the other instances.